material-overrides/assets/stylesheets/terminal.css

```diff
@@ -66,8 +66,19 @@
     content: attr(data-ty-prompt);
 }
 
+.md-typeset .table-termynal .md-typeset__table table th.center-header {
+    text-align: center;
+    padding-right: 22%;
+    width: 100%;
+}
+
 .md-typeset .table-termynal .md-typeset__table table th.left-header {
     background-color: unset;
-    border-bottom: .05rem solid var(--md-typeset-table-color);
+    border-top: .05rem solid var(--md-typeset-table-color);
     border-right: .05rem solid var(--md-typeset-table-color);
+    color: var(--md-typeset-table-color);
+}
+
+.md-typeset .table-termynal .md-typeset__table table td {
+    color: var(--md-typeset-table-color);
 }
```

Comment:
What if you try this:

```
.md-typeset .table-termynal table:not([class]) {
    border: none;
}

.md-typeset .table-termynal .md-typeset__table table th.left-header {
    border-bottom: .05rem solid rgba(255,255,255,.1);
    border-right: .05rem solid rgba(255,255,255,.1);
    border-color: rgba(255,255,255,.1);
    background-color: var(--black);
}

.md-typeset .table-termynal .md-typeset__table table th.center-header {
    text-align: center;
    width: 100%;
    color: var(--white);
    background-color: var(--pink);
    padding: 0 22% 0 0;
}

.md-typeset .table-termynal table:not([class]) td {
    background-color: var(--black);
    border-color: rgba(255,255,255,.1);
}
```

Reply:
It makes the table the same colors on light and dark mode (which in a terminal element, it probably should be this way).

I personally like the more subtle borders, but we can change it up - whatever works

Reply:
This does the trick 100%, LGTM!